e2e/archive/wallet/0.9.0/wallet.did

```diff
@@ -0,0 +1,282 @@
+type EventKind = variant {
```

Comment:
This should be in wallet/0.10.0/, since that is the version in which this will be released.